### PR TITLE
Clarify span status guidance for HTTP spans

### PR DESCRIPTION
**Pull request by arminru**
It was not clear if "status message" refers to the general span status description or the HTTP reason phrase (see [Gitter request](https://gitter.im/open-telemetry/opentelemetry-specification?at=5e831abbac3c3909b2dce63c)).

### CODE REVIEW

File: specification/data-http.md [hunk 1]
```diff
@@ -37,14 +37,14 @@ default span name.
...
+Implementations MUST set the [span status](api-tracing.md#status) if the HTTP communication failed
 or an HTTP error status code is returned (e.g. above 3xx).
 
 In the case of an HTTP redirect, the request should normally be considered successful,
 unless the client aborts following redirects due to hitting some limit (redirect loop).
-If following a (chain of) redirect(s) successfully, the Status should be set according to the result of the final HTTP request.
+If following a (chain of) redirect(s) successfully, the status should be set according to the result of the final HTTP request.
 
-Don't set a status message if the reason can be inferred from `http.status_code` and `http.status_text` already.
+Don't set a description on the span status if the reason can be inferred from `http.status_code` and `http.status_text` already.
```
Comment (Oberon00) [inline specification/data-http.md]:
```suggestion
Don't set a description of the span status if the reason can be inferred from `http.status_code` and `http.status_text` already.
```

Comment (carlosalberto) [reply specification/data-http.md]: What about `Don't set the span status description if ...`?

File: specification/data-http.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ In the case of an HTTP redirect, the request should normally be considered succe
 unless the client aborts following redirects due to hitting some limit (redirect loop).
 If following a (chain of) redirect(s) successfully, the status should be set according to the result of the final HTTP request.
 
-Don't set a description on the span status if the reason can be inferred from `http.status_code` and `http.status_text` already.
+Don't set the span status description if the reason can be inferred from `http.status_code` and `http.status_text` already.
```
Comment (jkwatson) [inline specification/data-http.md]: The word "already" isn't necessary. I'd remove it.

**Comment by bogdandrutu** [comment]
@arminru please try to follow the rules for contributing referenced here https://github.com/open-telemetry/opentelemetry-specification/blob/master/CONTRIBUTING.md#contributing and written here https://github.com/open-telemetry/community/blob/master/CONTRIBUTING.md#github-workflow :) Don't need to do anything now but for the future.

**Comment by arminru** [comment]
@bogdandrutu Sorry about that, I used the Edit✏️ feature right from the Github website which only allows me to commit on a branch since I have write access to the repo, otherwise it would've used my own fork. I thought it wouldn't matter since that's a change that I expected to merge rather instantly anyway. I'll use the more cumbersome manual approach on my local machine in future then to avoid this. 🙂

**Comment by Oberon00** [comment]
@bogdandrutu You should make a PR to add this caveat to CONTRIBUTING.md for people with write-access to take care of. I suspect there will be push-back however 😉

**Comment by bogdandrutu** [comment]
@Oberon00 I added the links there is already in the contributing reference to that

**Comment by carlosalberto** [comment]
Merged - this was a relatively minor clarification and had two approvals already (but had been standing forgotten for some time now ;( )

**Pull request merged**